tools/uniform_sampling.cpp

```diff
@@ -130,10 +158,47 @@ main (int argc, char** argv)
 
   // Parse the command line arguments for .pcd files
   vector<int> p_file_indices;
-  p_file_indices = parse_file_extension_argument (argc, argv, ".pcd");
+  vector<std::string> extension;
+  extension.push_back (".pcd");
+  extension.push_back (".ply");
+  extension.push_back (".vtk");
+  p_file_indices = parse_file_extension_argument (argc, argv, extension);
+
   if (p_file_indices.size () != 2)
```

Comment:
> if the ouput format at least is supported.

This line effectively checks that. If the format of the output file is not supported, then we won't have two file names parsed.

Reply:
You're right. So I'll make a proposition soon.